include/muu/integer.h

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+//
+
+#pragma once
+
+//! \file
+//! \brief Shorthands for fixed width integer types.
+
+#include "impl/std_type_traits.h"
```

Comment:
```suggestion
#include "fwd.h"
```

`fwd.h` is where the `<cstdint>` aliases are brought into the muu namespace.

It might actually be worth inverting that, though. Perhaps that belongs in another header (this one?)